libdnf/rpm/solv_sack_impl.hpp

```diff
@@ -104,6 +104,10 @@ class SolvSack::Impl {
 
     void make_provides_ready();
 
+    PackageId get_running_kernel() { return running_kernel; };
```

Comment:
"Const" and can be "noexcept". Or do we plan that the method may generate an exception in the future? 

`PackageId get_running_kernel() const noexcept { return running_kernel; };`

Reply:
The method is only getter and there is no detection. The detection is somewhere else. Added `const noexcept` and thanks for the suggestion.